database/javascript/2017/16138.yaml

```diff
@@ -0,0 +1,16 @@
+---
+cve: 2017-16138
+title: CVE in mime
+description: >
+    The mime module is vulnerable to regular expression denial of service when a mime lookup is performed on untrusted user input.
+cvss_v2: 5.0
+references:
+    - https://github.com/broofa/node-mime/issues/167
+    - https://nodesecurity.io/advisories/535
+
+affected:
+    - name: mime
+      version:
+        - "<1.4.1"
```

Comment:
This syntax is not supported by VictimsDB: https://github.com/victims/victims-cve-db#version-string-common

Reply:
Ah right.. forgot about this thing. Thanks for being thorough!